source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -157,5 +157,102 @@ the Query API:
+     - :manual:`$toDate </reference/operator/aggregation/toDate/>`     
+
+   * - `parseDate() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlString.html#parseDate()>`__
+     - :manual:`$dateFromString </reference/operator/aggregation/dateFromString/>`
+
+   * - `dayOfWeek() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlDate.html#dayOfWeek(com.mongodb.client.model.mql.MqlString)>`__
+     - :manual:`$dayOfWeek </reference/operator/aggregation/dayOfWeek/>`
+
+   * - `dayOfMonth() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlDate.html#dayOfMonth(com.mongodb.client.model.mql.MqlString)>`__
+     - :manual:`$dayOfMonth </reference/operator/aggregation/dayOfMonth/>`
+
+   * - `dayOfYear() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlDate.html#dayOfYear(com.mongodb.client.model.mql.MqlString)>`__
+     - :manual:`$dayOfYear </reference/operator/aggregation/dayOfYear/>`
+
+   * - `year() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlDate.html#year(com.mongodb.client.model.mql.MqlString)>`__
+     - :manual:`$year </reference/operator/aggregation/year/>`
+
+   * - `month() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlDate.html#month(com.mongodb.client.model.mql.MqlString)>`__
+     - :manual:`$month </reference/operator/aggregation/month/>`
+   
+   * - `week() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlDate.html#week(com.mongodb.client.model.mql.MqlString)>`__
+     - :manual:`$week </reference/operator/aggregation/week/>`
+
+   * - `hour() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlDate.html#hour(com.mongodb.client.model.mql.MqlString)>`__
+     - :manual:`$hour </reference/operator/aggregation/hour/>`
+
+   * - `minute() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlDate.html#minute(com.mongodb.client.model.mql.MqlString)>`__
+     - :manual:`$minute </reference/operator/aggregation/minute/>`
+
+   * - `second() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlDate.html#second(com.mongodb.client.model.mql.MqlString)>`__
+     - :manual:`$second </reference/operator/aggregation/second/>`
+
+   * - `millisecond() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlDate.html#millisecond(com.mongodb.client.model.mql.MqlString)>`__
+     - :manual:`$millisecond </reference/operator/aggregation/millisecond/>`
+
+.. include:: /includes/fundamentals/agg-mql-diff.rst
+
+Suppose you have some UTC time series data and need to
+match events that occurred on any Monday in 2018.
+
+To get the year from the ``date`` field, use the ``year()`` method, and to
+check if the year is ``2018``, use the ``eq()`` method. To get the day of
+the week (as a number), use the ``dayOfWeek()`` method, and to check if
+the day is Monday, corresponding to ``2``, use the ``eq()`` method.
```

Comment:
[suggestion]
This sentence reads a little long to me. Is it possible to break up into two, or would that be too many "to do X..." sentences?

```suggestion
check if the year is ``2018``, use the ``eq()`` method. To get the day of
the week (as a number), use the ``dayOfWeek()`` method. To check if
the day is Monday, corresponding to ``2``, use the ``eq()`` method.
```

Reply:
How is this revision?